src/components/Events/EventDetail/EventDetail.tsx

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from '@tanstack/react-query'
+import fetchEvent from 'api/fetch-event'
+const EventDetail = ({ eventId }: IProps) => {
+  const eventDetail = useQuery(['todos'], () => fetchEvent({ id: eventId }))
```

Comment:
I forgot to mention this query name should be `event` not `todos` I believe we get that from the React Query documentation

Reply:
aight! chenge made